scripts/zones/Lebros_Cavern/mobs/Qiqirn_Mine.lua

```diff
@@ -0,0 +1,52 @@
+-----------------------------------
+-- Area: Lebros Cavern (Excavation Duty)
+-- MOB: Qiqirn Mine
+-----------------------------------
+local ID = require("scripts/zones/Lebros_Cavern/IDs")
+require("scripts/globals/allyassist")
+require("scripts/globals/status")
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:setMobMod(dsp.mobMod.NO_MOVE, 1)
+    mob:SetAutoAttackEnabled(false)
+end
+
+function onMobRoam(mob)
+    dsp.ally.startAssist(mob, dsp.ally.ASSIST_PLAYER);
+end;
+
+function onMobFight(mob,target)
+    local instance = mob:getInstance()
+    local players = instance:getChars();
+
+    for i,v in pairs(players) do
+        if mob:getLocalVar("Time") == 0 then
+            v:messageSpecial(ID.text.MINE_COUNTDOWN,10)
+            mob:setLocalVar("Time",1)
+        elseif mob:getLocalVar("Time") == 1 then
+            mob:timer(5000, function(mob) v:messageSpecial(ID.text.MINE_COUNTDOWN,5) end)
+            mob:setLocalVar("Time",2)
+        elseif mob:getLocalVar("Time") == 2 then
+            mob:timer(6000, function(mob) v:messageSpecial(ID.text.MINE_COUNTDOWN,4) end)
+            mob:setLocalVar("Time",3)
+        elseif mob:getLocalVar("Time") == 3 then
+            mob:timer(7000, function(mob) v:messageSpecial(ID.text.MINE_COUNTDOWN,3) end)
+            mob:setLocalVar("Time",4)
+        elseif mob:getLocalVar("Time") == 4 then
+            mob:timer(8000, function(mob) v:messageSpecial(ID.text.MINE_COUNTDOWN,2) end)
+            mob:setLocalVar("Time",5)
+        elseif mob:getLocalVar("Time") == 5 then
+            mob:timer(9000, function(mob) v:messageSpecial(ID.text.MINE_COUNTDOWN,1) end)
+            mob:setLocalVar("Time",6)
+        elseif mob:getLocalVar("Time") == 6 then
+            mob:timer(10000, function(mob) mob:useMobAbility(1838) end)
+            mob:setLocalVar("Time",7)
+        elseif mob:getLocalVar("Time") == 7 then
+            mob:timer(12000, function(mob) mob:setHP(0) end)
```

Comment:
`Time` doesn't extend past 7; will this be setting up a lot of `setHP()` calls after `Time` has reached 7, but `onMobFight()` keeps getting called?

Reply:
Once hit hits hp 0 it dies and despawns. Voiding the onmobfight. Next bomb set will start counter over again. Wash repeat

Reply:
From what I understand, `onMobFight` gets called every combat tick, and lua timers are non-blocking~! So until the mob actually dies (12 seconds after spawn), a lot of non-blocking timers to invoke `setHP` will be set up (since Time never gets set to 8 or anything).

Reply:
Yea i can easily add another var set